From 38220cc2e97215588b662584d6ad84d63415842f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ond=C5=99ej=20Lyson=C4=9Bk?= <olysonek@redhat.com>
Date: Thu, 20 Sep 2018 21:17:40 +0200
Subject: [PATCH 094/117] pwmconfig: Fix a sed expression
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This fixes commit 3daae37d58cd.
The escaping inside backticks is really kinda crazy.

Fixes #126

Signed-off-by: Ondřej Lysoněk <olysonek@redhat.com>
---
 prog/pwm/pwmconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/prog/pwm/pwmconfig b/prog/pwm/pwmconfig
index c24c891..a40acd5 100755
--- a/prog/pwm/pwmconfig
+++ b/prog/pwm/pwmconfig
@@ -879,7 +879,7 @@ select pwms in $pwmactive "Change INTERVAL" "Just quit" "Save and quit" "Show co
 		break ;;
 
 	"$(echo ${pwmactive} |sed -e 's/ /\n/g' | egrep "${pwms}")" )
-		pwmsed=$(echo ${pwms} | sed -e 's/\//\\\\\//g') #escape / for sed
+		pwmsed=$(echo ${pwms} | sed -e 's/\//\\\//g') #escape / for sed
 		echo
 
 		echo 'Devices:'
-- 
1.8.3.1

